src/EditorFeatures/Test/StackTraceExplorer/StackTraceExplorerTests.cs

```diff
@@ -170,7 +231,130 @@ class MyClass
 }");
         }
 
-        [Fact(Skip = "The parser does not handle arity on types yet")]
+        [Fact]
+        public Task TestSymbolFound_ParameterSpacing()
+        {
+            return TestSymbolFoundAsync(
+                "at ConsoleApp.MyClass.M( String   s    )",
+                @"
+namespace ConsoleApp
+{
+    class MyClass
+    {
+        void [|M|](string s)
+        {
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_OverloadsWithSameName()
+        {
+            return TestSymbolFoundAsync(
+                "at ConsoleApp.MyClass.M(String value)",
+                @"
+namespace ConsoleApp
+{
+    class MyClass
+    {
+        void [|M|](string value)
+        {
+        }
+
+        void M(int value)
+        {
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_ArrayParameter()
+        {
+            return TestSymbolFoundAsync(
+                "at ConsoleApp.MyClass.M(String[] s)",
+                @"
+namespace ConsoleApp
+{
+    class MyClass
+    {
+        void [|M|](string[] s)
+        {
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_MultidimensionArrayParameter()
+        {
+            return TestSymbolFoundAsync(
+                "at ConsoleApp.MyClass.M(String[,] s)",
+                @"
+namespace ConsoleApp
+{
+    class MyClass
+    {
+        void [|M|](string[,] s)
+        {
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_MultidimensionArrayParameter_WithSpaces()
+        {
+            return TestSymbolFoundAsync(
+                "at ConsoleApp.MyClass.M(String[ , ] s)",
+                @"
+namespace ConsoleApp
+{
+    class MyClass
+    {
+        void [|M|](string[,] s)
+        {
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_MultidimensionArrayParameter_WithSpaces2()
+        {
+            return TestSymbolFoundAsync(
+                "at ConsoleApp.MyClass.M(String[,] s)",
+                @"
+namespace ConsoleApp
+{
+    class MyClass
+    {
+        void [|M|](string[ , ] s)
+        {
+        }
+    }
+}");
+        }
+
+        [Fact]
+        public Task TestSymbolFound_MultidimensionArrayParameter2()
+        {
+            return TestSymbolFoundAsync(
+                "at ConsoleApp.MyClass.M(String[,][] s)",
+                @"
+namespace ConsoleApp
+{
+    class MyClass
```

Comment:
do we have tests with generic types?

Reply:
There was one for nested that are disabled. I'll add a few just for generics on their own